Assets/HoloToolkit/Input/Scripts/Cursor/CursorModifier.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity.InputModule
+{
+    /// <summary>
+    /// Component that can be added to any collider to modify how a cursor reacts when on that collider.
+    /// </summary>
+    public class CursorModifier : MonoBehaviour, ICursorModifier
```

Comment:
Yes, that's what the comment says. More specifically, it needs to be added to a game object with a collider